### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  root: true,
+  extends: ['eslint-config-digitalbazaar'],
+  env: {
+    node: true
+  },
+  globals: {
+    Uint8Array: true
```
Comment (davidlehn) [inline .eslintrc.js]: Is this needed?

File: Bitstring.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  set(position, on) {
+    if(typeof on !== 'boolean') {
+      throw new TypeError('"on" must be a boolean.');
+    }
+    const {index, bit} = _parsePosition(position, this.length);
+    if(on) {
+      this.bits[index] |= bit;
+    } else {
+      this.bits[index] &= 0xFFFFFFFF ^ bit;
```
Comment (davidlehn) [inline Bitstring.js]: Why not just 0xFF?

File: .eslintrc.js (hunk 1 above)
Comment (dlongley) [reply .eslintrc.js]: Apparently not! Removed.

File: Bitstring.js (hunk 2 above)
Comment (dlongley) [reply Bitstring.js]: Because I was doing int things before and missed this, thanks! Fixed.

**Pull request merged**